docs/configuration/config-file-reference.md

```diff
@@ -104,6 +104,20 @@ Where default_value is the value to use if the environment variable is undefined
 # The table_manager_config configures the Cortex table-manager.
 [table_manager: <table_manager_config>]
 
+data_purger:
```

Comment:
We try to keep consistency between YAML config and CLI flags. The YAML config block is called `data_purger` while CLI flags have the prefix `-purger.`. Have you considered renaming the YAML config block from `data_purger` to `purger`?

Reply:
Right, I missed doing that somehow. Pushed the changes. Thanks!